soh/soh/Enhancements/custom-message/CustomMessageManager.cpp

```diff
@@ -623,6 +623,29 @@ void CustomMessage::InsertNumber(uint8_t num) {
     Replace("[[d]]", std::to_string(num));
 }
 
+void CustomMessage::SetSingularPlural() {
+    for (std::string& str : messages) {
+        size_t firstBar = str.find('|');
+        if (firstBar != std::string::npos) {
+            size_t euroSign = str.find('€');
```

Comment:
```suggestion
            size_t euroSign = str.find("€");
```
needed this to build on my system